kbase-extension/static/kbase/js/widgets/appWidgets2/input/dynamicDropdownInput.js

```diff
@@ -170,6 +171,11 @@ define([
 
         function validate() {
             return Promise.try(() => {
+                if (model.exactMatchError) {
```

Comment:
If we're in a state where we have an exactMatchError, then force the validation to fail and put in a custom, helpful (hopefully?) error message.